build.gradle.kts

```diff
@@ -0,0 +1,202 @@
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import org.apache.tools.ant.filters.ReplaceTokens
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+plugins {
+    id("eclipse")
+    id("idea")
+    id("net.neoforged.gradle.userdev") version "7.+"
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
+    }
+}
+
+val mod_version: String by project.properties
+val minecraft_version: String by project.properties
+val neo_version: String by project.properties
+val mod_id: String by project.properties
+
+version = mod_version
+val archivesBaseName = "ICBM-classic-${minecraft_version}"
+
+base {
+    archivesName = mod_id
+}
+
+//fancyGradle {
```

Comment:
Can remove fancyGradle commented out lines, its only need to fix forge's broken 1.12 builds.